README.md

```diff
@@ -1,134 +1,32 @@
-# docker-stacks
+[![Google Group](https://img.shields.io/badge/-Google%20Group-lightgrey.svg)](https://groups.google.com/forum/#!forum/jupyter)
+![Read the Docs](https://img.shields.io/readthedocs/jupyter-docker-stacks.svg)
+[![](https://images.microbadger.com/badges/version/jupyter/base-notebook.svg)](https://microbadger.com/images/jupyter/base-notebook "Get your own version badge on microbadger.com")
 
-[![Build Status](https://travis-ci.org/jupyter/docker-stacks.svg?branch=master)](https://travis-ci.org/jupyter/docker-stacks)
-[![Join the chat at https://gitter.im/jupyter/jupyter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/jupyter/jupyter?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Jupyter Docker Stacks
 
-Opinionated stacks of ready-to-run Jupyter applications in Docker.
+Jupyter Docker Stacks are a set of ready-to-run Docker images containing Jupyter applications and interactive computing tools.
 
 ## Quick Start
 
-If you're familiar with Docker, have it configured, and know exactly what you'd like to run, one of these commands should get you up and running:
+The two examples below may help you get started if you [have Docker installed](https://docs.docker.com/install/) know [which Docker image](http://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html) you want to use, and want to launch a single Jupyter Notebook server in a container.
 
-```
-# Run an ephemeral Jupyter Notebook server in a Docker container in the terminal foreground.
-# Note that any work saved in the container will be lost when it is destroyed with this config.
-# -ti: pseudo-TTY+STDIN open.
-# -rm: remove the container on exit.
-# -p: publish port to the host
-docker run -ti --rm -p 8888:8888 jupyter/<your desired stack>:<git-sha-tag>
+The [User Guide on ReadTheDocs](http://jupyter-docker-stacks.readthedocs.io/) describes additional uses and features in detail.
 
-# Run a Jupyter Notebook server in a Docker container in the terminal foreground.
-# Any files written to ~/work in the container will be saved to the current working
-# directory on the host.
-docker run -ti --rm -p 8888:8888 -v "$PWD":/home/jovyan/work jupyter/<your desired stack>:<git-sha-tag>
+**Example 1:** This command pulls the `jupyter/scipy-notebook` image tagged `2c80cf3537ca` from Docker Hub if it is not already present on the local host. It then starts a container running a Jupyter Notebook server and exposes the server on host port 8888. The server logs appear in the terminal and include a URL to the notebook server. The container remains intact for restart after notebook server exit.::
```

Comment:
If markdown, the double colon at end is unnecessary. One can be removed.

Reply:
Good catch. Copy/paste error from the index.rst on the docs site.